index.js

```diff
@@ -16,14 +16,16 @@ module.exports = {
 
       defaultConfig: {
         archivePath: path.join('tmp', 'deploy-archive'),
-        archiveName: 'build.tar'
+        archiveName: 'build.tar',
+        addToDistFiles: false
       },
 
       didBuild: function(context) {
         var self = this;
         var archivePath = this.readConfig('archivePath');
         var archiveName = this.readConfig('archiveName');
-        this.distDir = context.distDir;
+        var addToDistFiles = this.readConfig('addToDistFiles');
+        this.distDir = this.readConfig('distDir') || context.distDir;
```

Comment:
We might as well make `distDir` configurable like other plugins... then we can simply rely on `this.readConfig('distDir')`.
https://github.com/ember-cli-deploy/ember-cli-deploy-s3/blob/master/index.js#L19-L21